src/resource/event/event.controller.ts

```diff
@@ -74,7 +74,7 @@ export class EventController {
     @Get(':id')
     @GetEventByIdApi()
     async getEventById(@Param('id') eventId: number): Promise<ReturnEventDTO> {
-        return await this.eventService.fetchById(eventId);
+        return await this.eventService.fetch(eventId);
```

Comment:
fetchById because we don't know if we fetch all or not.
You could use [serviceName].fetchAll or fetchOne and albo fetchById